src/source/ObjcppGenerator.scala

```diff
@@ -138,6 +138,10 @@ class ObjcppGenerator(spec: Spec) extends BaseObjcGenerator(spec) {
       val objcSelf = if (i.ext.objc && i.ext.cpp) self + "CppProxy" else self
 
       if (i.ext.cpp) {
+		// Disable warnings in Clang about directly accessing the ivars of an object
+		w.wl("#if __clang__")
+        w.wl("#pragma clang diagnostic ignored \"-Wdirect-ivar-access\"")
+		w.wl("#endif")
```

Comment:
The generated code doesn't include the #if/#endif here, so I presume it wasn't regenerated after the most recent change?  Please re-generate to reflect the latest state of the generator.  After seeing the commentary I'm fine with merging this either with or without the #if.